pom.xml

```diff
@@ -121,6 +121,14 @@
 			<url>https://oss.sonatype.org/content/repositories/snapshots</url>
 		</repository>
 		<repository>
+			<id>jitpack.io</id>
```

Comment:
Probably can be removed now. Before we had our own repo with bliki which I guess it is the reason why jitpack was introduced